docs/manual/src/README.md

```diff
@@ -0,0 +1,57 @@
+# The docs for the "manual".
+
+The manual is what we publish to https://mozilla.github.io/uniffi-rs/ and
+is generated from the content in this directory.
+
+This README is not published as part of the docs - but almost everything
+else in this directory is. This document describes how that publishing
+works.
+
+* This repo uses `mkdocs` - but **we must never deploy using that tool**.
+  Deploying using mkdocs will publish into the root of the site, **damaging
+ the versioning system we have in place.**
+
+* We use a tool, [`mike`](https://github.com/jimporter/mike), which manages
+  the `mkdocs` build process and the deployment - it deploys to a versioned
+  (eg, `./0.27`) directory and manages aliases (eg, `latest`) on the site.
+
+* There are various HTML redirects and symlinks keeping things together. These
+  are maintained by hand. See also any docs related scripts in `./tools`.
+
+To put it another way: we never deploy the entire site, but instead use `mike`
+to deploy just a single version.
+
+## How to deploy
+
+* Prepare your Python environment by executing `pip install -r tools/requirements_docs.txt`
+
+* Although done by CI, we publish `main` as a version called `next` - so you 
+  could deploy this with `mike deploy next --push`
```

Comment:
I don't understand this. Is this done automatically or is it not?
If it is, it shouldn't be in this section of the instructions. Let's focus on the manual steps that _need_ to be done.

Reply:
Yes, pushing docs for the main branch is on CI, pushing docs for a just-released version is not. But sure, I'll remove this.